schemas/v1.0/schema.yaml

```
@@ -0,0 +1,563 @@
+$id: 'https://spec.openapis.org/arazzo/1.0/schema/2024-08-01'
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+description: |-
+  The description of OpenAPI Initiative Arazzo v1.0.0 documents
+  without schema validation, as defined by https://spec.openapis.org/arazzo/v1.0.0
+type: object
+properties:
+  arazzo:
+    description: The version number of the Arazzo Specification
+    type: string
+    pattern: '^1\.\d+\.\d+(-.+)?$'
+  info:
+    $ref: '#/$defs/info'
+  sourceDescriptions:
+    description: A list of source descriptions such as Arazzo or OpenAPI
+    type: array
+    uniqueItems: true
+    minItems: 1
+    items:
+      $ref: '#/$defs/source-description-object'
+  workflows:
+    description: A list of workflows
+    type: array
+    uniqueItems: true
+    minItems: 1
+    items:
+      $ref: '#/$defs/workflow-object'
+  components:
+    $ref: '#/$defs/components-object'
+required:
+  - arazzo
+  - info
+  - sourceDescriptions
+  - workflows
+$ref: '#/$defs/specification-extensions'
+unevaluatedProperties: false
+$defs:
+  info:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#info-object
+    description: Provides metadata about the Arazzo description
+    type: object
+    properties:
+      title:
+        description: A human readable title of the Arazzo Description
+        type: string
+      summary:
+        description: A short summary of the Arazzo Description
+        type: string
+      description:
+        description: A description of the purpose of the workflows defined. CommonMark syntax MAY be used for rich text representation
+        type: string
+      version:
+        description: The version identifier of the Arazzo document (which is distinct from the Arazzo Specification version)
+        type: string
+    required:
+      - title
+      - version
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  source-description-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#source-description-object
+    description: |-
+      Describes a source description (such as an OpenAPI description)
+      that will be referenced by one or more workflows described within
+      an Arazzo description
+    type: object
+    properties:
+      name:
+        description: A unique name for the source description
+        type: string
+        pattern: '^[A-Za-z0-9_\-]+$'
+      url:
+        description: A URL to a source description to be used by a workflow
+        type: string
+        format: uri-reference
+      type:
+        description: The type of source description
+        enum:
+          - arazzo
+          - openapi
+    required:
+      - name
+      - type
```

```suggestion
      - url
```
It's `name` and `url` that are the required properties.